.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npm run precommit
```

Comment:
I created the hook following husky instructions (npx approach) and this is what was created - https://github.com/developerforce/lwc-builder/blob/main/.husky/pre-commit, not sure why yours is different?

Reply:
I've said nothing, it's an issue with yarn and windows so no worries - https://typicode.github.io/husky/#/?id=yarn-on-windows